src/server/active_tran_server.cpp

```diff
@@ -276,8 +276,17 @@ void
 active_tran_server::disconnect_page_server ()
 {
   assert_is_active_tran_server ();
+  std::string msg = "" + static_cast<int> (cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER);
```

Comment:
Send the memory reprezentation of cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER.
```suggestion
  const int payload = cubcomm::server_server::CONNECT_ACTIVE_TRAN_TO_PAGE_SERVER;
  std::string msg (reinterpret_cast<const char *> (&payload), sizeof (payload));
```